building/validate-apt-branch.sh

```diff
@@ -0,0 +1,14 @@
+if [ -z "${HOMEWORLD_APT_BRANCH:-}" ]
```

Comment:
It might be worth sticking these two script in a subdirectory of `/building/`, as directly under `/building/` is a strange place to stick scripts.